tools/ci-scripts/trigger-build.sh

```diff
@@ -6,7 +6,8 @@ then
   git clone https://github.com/"$DOCKER_PUSH_REPO".git docker-push-repo
 
   cd docker-push-repo/freecodecamp
-  git pull --rebase origin master
+  git submodule update --init --remote --recursive
```

Comment:
This works locally from the root of the repository

Reply:
Yes, I just realised that.